tsl/test/shared/expected/gapfill-15.out

```diff
@@ -3490,3 +3490,72 @@ FROM (SELECT NULL::timestamptz AS time LIMIT 0) s GROUP BY 1;
 (3 rows)
 
 RESET timezone;
+CREATE TABLE uuid_events (id UUID NOT NULL, device INT, temp FLOAT);
+SELECT table_name FROM create_hypertable('uuid_events', 'id');
+ table_name  
+ uuid_events
+(1 row)
+
+INSERT INTO uuid_events VALUES
+       ('0194214e-cd00-7000-a9a7-63f1416dab45', 2, 2.0),
+       ('01942117-de80-7000-8121-f12b2b69dd96', 1, 1.0),
+       ('01942bd2-7380-7000-9bc4-5f97443907b8', 5, 5.0),
+       ('01942d52-f900-7000-866e-07d6404d53c1', 6, 6.0);
+SELECT _timescaledb_functions.timestamptz_from_uuid_v7(id)
+FROM uuid_events;
+   timestamptz_from_uuid_v7   
+ Wed Jan 01 02:00:00 2025 PST
+ Wed Jan 01 01:00:00 2025 PST
+ Fri Jan 03 03:00:00 2025 PST
+ Fri Jan 03 10:00:00 2025 PST
+(4 rows)
+
+\set ON_ERROR_STOP 0
+SELECT time_bucket_gapfill('1 day',  id) AS day,
+    locf(avg(temp)) AS temp
+    FROM uuid_events
+    WHERE id > _timescaledb_functions.uuid_v7_from_timestamptz_zeroed('2025-01-01 PST') AND
+          id <= _timescaledb_functions.uuid_v7_from_timestamptz_zeroed('2025-01-04 PST')
+    GROUP BY day
+    ORDER BY day DESC;
+ERROR:  function time_bucket_gapfill(unknown, uuid) does not exist
```

Comment:
I am not sure myself of the value of having a test to show that a function doesn't exist. Should we have this?